tracer/src/Datadog.Trace/TracerConstants.cs

```diff
@@ -15,5 +15,6 @@ internal static class TracerConstants
         public const ulong MaxTraceId = 9_223_372_036_854_775_807;
 
         public const string AssemblyVersion = "2.18.0.0";
```

Comment:
Maybe set this as `ThreePartVersion + ".0"`?

Reply:
Not keen, the string contact would add startup overhead :)